lucene/core/src/java/org/apache/lucene/codecs/compressing/Lucene90CompressingStoredFieldsFormat.java

```diff
@@ -40,7 +40,7 @@
  *
  * @lucene.experimental
  */
-public class CompressingStoredFieldsFormat extends StoredFieldsFormat {
+public class Lucene90CompressingStoredFieldsFormat extends StoredFieldsFormat {
```

Comment:
I think these should be moved to the package `lucene90.compressing`.

Reply:
done